### ISSUE TITLE
[BUG] read_redcap failing after redcap upgrade

### ISSUE THREAD

**Issue by johnryan412** [CHOP-CGTInformatics/REDCapTidieR#191]
# Expected Behavior
Import my redcap dataset from our institution's URL using the API token.  Over the weekend our institution pushed a redcap system migration.   "System migration has been completed. Please contact [hs.appsupport@pitt.edu](hs.appsupport@pitt.edu) with any issues, being as detailed as possible to allow quick investigation.
 
The URL has changed, there is a redirect in place from the old URL (ctsiredcap.pitt.edu) to the new one (redcap-std.hs.pitt.edu). This will allow existing survey invitations and links to function as normal. All new invitations sent will go through the new URL."

# Current Behavior
```

> tidy_data<-read_redcap(redcap_url, token)
Error in `read_redcap()`:
✖ The REDCapR export operation was not successful.
! An unexpected error occured.
ℹ This means that you probably discovered a bug!
Caused by error in `curl::curl_fetch_memory()`:
! Maximum (10) redirects followed
Backtrace:
  1. REDCapTidieR::read_redcap(redcap_uri, token)
 10. REDCapR::redcap_metadata_read(...)
 11. REDCapR:::kernel_api(redcap_uri, post_body, config_options)
 12. httr::POST(...)
 13. httr:::request_perform(req, hu$handle$handle)
 15. httr:::request_fetch.write_memory(req$output, req$url, handle)
 16. curl::curl_fetch_memory(url, handle = handle)

```
## How to Reproduce the Bug:
redcap_uri<-"https://ctsiredcap.pitt.edu"
token <- Sys.getenv("api")
tidy_data<-read_redcap(redcap_uri, token)

(I have verified that the token is still correct, and have tried the new URL)

**Screenshots**

## Failure Logs

If applicable, submit any failure logs or error messages here.

### Checklist
Before submitting this issue, please check and verify below that the submission meets the below criteria:

- [ ] The issue is atomic
- [ ] The issue description is documented
- [ ] The issue title describes the problem succinctly
- [ ] Developers are assigned to the issue
- [ ] Labels are assigned to the issue

**Comment by ezraporter** [comment]
Thanks for the bug report! Unfortunately this looks to be a low-level issue with how R is interacting with your institution's REDCap server so it won't be something `REDCapTidieR` can fix.

You can try filing a bug report with `REDCapR`, the library we depend on to interact with REDCap: https://github.com/OuhscBbmc/REDCapR/issues. Your best bet may be to work with your institution's REDCap admins to resolve the excessive redirects issue in the error message you posted.

I'm going to close this issue for now but feel free to re-open if, for example, there's some option `REDCapTidieR` can expose that will let you get around the redirect error.

**Comment by ezraporter** [comment]
Should also note it looks like you may be using the old url in your code example? Switching to the new url may resolve.

**Comment by johnryan412** [comment]
Yeah, unfortunately no luck with either.  I've reached out to our IT team.
Will provide an update if we find a resolution.

**Comment by johnryan412** [comment]
It was an issue with the link - There is now a special link for API access.  Thanks for your help!

FYI - anyone from Pitt, it's https://redcap-std.hs.pitt.edu/redcap/api/
